styles/components/MyAccount.module.scss

```diff
@@ -31,11 +31,14 @@
     margin-top: var(--nypl-space-l);
     display: flex;
     flex-direction: row;
-    @media (max-width: 600) {
+    @media screen and (max-width: 600px) {
```

Comment:
Can you switch these over to use `min-width` instead? That'll follow the "mobile-first" approach. So in this case it'll be `flex-direction: column` be default and once the `min-width: 600px` value kicks in, it'll switch to `flex-direction: row`.

You can also use `var(--nypl-breakpoint-medium)` as the CSS var.